src/vm/excep.cpp

```diff
@@ -5152,10 +5138,20 @@ LONG InternalUnhandledExceptionFilter(
 
 } // LONG InternalUnhandledExceptionFilter()
 
+static bool s_useEntryPointFilter = false;
+
+void ParseUseEntryPointFilter(LPCWSTR value)
+{
+#ifdef PLATFORM_WINDOWS // This feature has only been tested on Windows, keep it disabled on other platforms
```

Comment:
Is the code under the ifdef actually compiled on Windows? I do not think `PLATFORM_WINDOWS` is defined for the C/C++ anywhere today.

Reply:
@jkotas This must be defined on Windows.  I put an #error inside the block to get it to fail compile to check for sure.  It is also used in a few other files in the VM eepolicy.cpp & peimagelayout.cpp.

Reply:
Yes, it is fixed now. @elinor-fung added `PLATFORM_WINDOWS` to cmake scripts a few days ago.